nautobot/ipam/factory.py

```diff
@@ -438,16 +438,16 @@ class Params:
         has_vrf = factory.Faker("pybool")
         is_container = factory.Faker("pybool")
         is_ipv6 = factory.Faker("pybool")
-        ipv6_cidr = factory.Faker("ipv6", network=True)
+        ipv6_cidr = UniqueFaker("ipv6", network=True)
```

Comment:
Actually, this doesn't seem like it's doing what it appears to do at a glance - since we're replacing the netmask provided by `ipv6_cidr` with a new random netmask, this could convert both `2001::1/128` and `2001::2/128` into say `2000::/64` and we'd still have a collision, right?

Reply:
My guess is the cidr gets picked first so you have a 1/128 chance of getting `::/0` instead of a 1/2^128 chance like you'd expect. 

See comment below regarding the prefix length

Reply:
Thanks for the clarification. :-)